osp/osp.c

```diff
@@ -462,13 +462,27 @@ osp_delete_scan (osp_connection_t *connection, const char *scan_id)
  */
 int
 osp_get_performance_ext (osp_connection_t *connection,
-                         osp_get_performance_opts_t opts,
-                         char **graph, char **error)
+                         osp_get_performance_opts_t opts, char **graph,
+                         char **error)
 {
   entity_t entity;
   int rc;
+  time_t now;
+
+  if (!connection)
+    return 1;
```

Comment:
I think this is slightly dangerous because it does not set "error".  Also would be more
consistent if it set "error" because all the other fail exits do.